test/functional/wallet_importdescriptors.py

```diff
@@ -102,6 +102,20 @@ def test_import_unused_key_existing(self):
                              wallet=wallet)
         wallet.unloadwallet()
 
+    def test_import_unused_noprivs(self):
+        self.log.info("Test import of unused(KEY) to wallet without privkeys")
+        self.nodes[0].createwallet(wallet_name="import_unused_noprivs", disable_private_keys=True)
+        wallet = self.nodes[0].get_wallet_rpc("import_unused_noprivs")
+
+        xpub = "tpubD6NzVbkrYhZ4YNXVQbNhMK1WqguFsUXceaVJKbmno2aZ3B6QfbMeraaYvnBSGpV3vxLyTTK9DYT1yoEck4XUScMzXoQ2U2oSmE2JyMedq3H"
+        self.test_importdesc({"timestamp": "now", "desc": descsum_create(f"unused({xpub})")},
+                             success=False,
+                             error_code=-4,
+                             error_message="Cannot import unused() to wallet without private keys enabled",
```

Comment:
In 61001085941fd6d78f4cd46c9b3d65914b6e961e "wallet, rpc: Disallow importing unused() to wallets without privkeys"

This doesn't allow to import unused descriptors with xpubs. `addhdkeys` also doesn't allow unused with xpubs.
Is there any use of unused descriptors with xpub? If not, then perhaps we should disable that while parsing the unused descriptor?